tests/runtest.sh

```diff
@@ -383,7 +383,10 @@ function create_core_overlay {
     cp -f -v "$coreFxNativeBinDir/Native/"*."$libExtension" "$coreOverlayDir/" 2>/dev/null
 
     cp -f -v "$coreClrBinDir/"* "$coreOverlayDir/" 2>/dev/null
-    cp -f -v "$mscorlibDir/mscorlib.dll" "$coreOverlayDir/"
+    cp -f -v "$mscorlibDir/mscorlib.dll" "$coreOverlayDir/" 2>/dev/null
+    if [ -d "$mscorlibDir" ] && [ -d "$mscorlibDir/bin" ]; then
```

Comment:
Check for "$mscorlibDir" is not necessary, as a file under it is being used a line above